config-service/build.gradle.kts

```diff
@@ -10,8 +10,34 @@ plugins {
 
 dependencies {
   implementation(project(":config-service-impl"))
+
+  constraints {
+    implementation("com.google.guava:guava:30.0-jre") {
+      because("https://snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-1015415")
+    }
+    runtimeOnly("io.netty:netty-codec-http2:4.1.53.Final") {
+      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
+    }
+    runtimeOnly("io.netty:netty-handler-proxy:4.1.53.Final") {
+      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
+    }
+  }
+
+  implementation("org.hypertrace.core.grpcutils:grpc-server-utils:0.2.0")
```

Comment:
There's some old versions in here. Since it's a new service, can you audit them and bring them all up to latest.

Reply:
Updated